src/utils/schemas/normalized/fishingRoom.json

```diff
@@ -0,0 +1,27 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "FishingRoom",
+  "type": "object",
+  "properties": {
+    "type": {
+      "const": "fishingRoom"
+    },
+    "playerState": {
+      "enum": [
+        "AtEntrance",
+        "BrokenLine",
+        "Caught",
+        "Fishing"
+      ]
+    },
+    "brokenRod": {
```

Comment:
nit: How about "isRodBroken"? I like when boolean names look like a question. I could interpret this as a rod object that is broken.

I'll leave it up to you

Reply:
`isRodBroken` sounds good to me. Updated.